main.css

```diff
@@ -1089,6 +1089,11 @@ input[type=range]:focus::-ms-fill-upper {
 	z-index: 3;
 }
 
+#popupSelector .title {
+    margin-bottom: 10px;
+    font-weight: bold;
+}
+

```

Comment:
Are you sure, they are indented consistently?

![image](https://user-images.githubusercontent.com/69431456/113569301-e2585080-962f-11eb-85aa-02dd785b7618.png)